aptos-move/framework/aptos-framework/sources/GovernanceProposal.move

```diff
@@ -3,16 +3,44 @@
 /// This is separate from the AptosGovernance module to avoid circular dependency between AptosGovernance and Stake.
 module AptosFramework::GovernanceProposal {
     friend AptosFramework::AptosGovernance;
+    use Std::ASCII::{String, string};
 
-    struct GovernanceProposal has store, drop {}
+    struct GovernanceProposal has store, drop {
+        /// The location (e.g. url) where the proposal resolution script's code can be accessed.
+        /// Maximum length allowed is 256 chars.
+        code_location: String,
+        /// Description of the proposal.
```

Comment:
comment should probably be `title of the proposal`?

Reply:
Good catch. Updated.